apps/blog/src/pages/index.astro

```diff
@@ -17,33 +17,63 @@ const posts = [
     title: post.data.title,
     date: new Date(post.data.date),
     image: post.data.image,
+    authors: post.data.authors || [],
+    category: (post as any).data.category,
   })),
   ...pages.map((page) => ({
     url: `${prefix === "/" ? "" : prefix}${page.url}`,
     title: (page as any).data.title,
     date: new Date((page as any).data.date),
     image: (page as any).data.image,
+    authors: (page as any).data.authors || [],
+    category: (page as any).data.category,
   })),
 ];
 posts.sort((a, b) => b.date.getTime() - a.date.getTime());
+
+function capitalizeFirstLetter(word: string) {
+  return word.charAt(0).toUpperCase() + word.slice(1);
+}
+
+function formatDate(date: Date) {
```

Comment:
There already exists a formatDate function in post-markdown.astro:11, this is also a wrong date format.

Just extract it somewhere.